events/views.py

```diff
@@ -1,3 +1,28 @@
-# from django.shortcuts import render
+from django.shortcuts import render, redirect
+from events.forms import EventCreationForm
+from django.contrib import messages
+from events.class_models.participant_model import EventParticipant
 
-# Create your views here.
+
+def create_event(request):
+    if not request.user.is_authenticated:
+        return redirect("welcome_page")
+
+    if request.method == 'POST':
+        print(request.POST)
+        form = EventCreationForm(request.POST)
+        if form.is_valid():
+            print("Valid form")
+            new_event = form.save()
+
+            event_participant = EventParticipant(event_id=new_event, user_id=request.user, is_creator=True)
+            event_participant.save()
```

Comment:
Maybe you can override the save method of the form and save the event participant there.

Reply:
done, override the form save method